i18n/fr/code.json

```diff
@@ -0,0 +1,432 @@
+{"Get started!": {
```

Comment:
Where is this file used?

Reply:
This file is used to change the text/Strings in the index.js file inside the src folder from English to French

Reply:
This file was made with the help of docusaurus i18n translation api